provider/resource_keycloak_openid_user_attribute_protocol_mapper.go

```diff
@@ -84,6 +84,12 @@ func resourceKeycloakOpenIdUserAttributeProtocolMapper() *schema.Resource {
 				Default:      "String",
 				ValidateFunc: validation.StringInSlice([]string{"JSON", "String", "long", "int", "boolean"}, true),
 			},
+			"aggregate_attrs": {
```

Comment:
this is a nitpick, but I'd prefer if this was called `aggregate_attributes`

Reply:
ok, done